### PR TITLE
Fix & upgrade

### PR DESCRIPTION
**Pull request by hannesbe**
Broken download URLs fixed & upgraded to 1.8.8

### CODE REVIEW

File: manual/synergy/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿$packageName= 'synergy'
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
-$url   = "http://synergy-project.org/files/packages/synergy-v1.7.6-stable-bcb9da8-Windows-x86.msi"
-$url64 = "http://synergy-project.org/files/packages/synergy-v1.7.6-stable-bcb9da8-Windows-x64.msi"
+$url   = "https://github.com/afzaalace/synergy-stable-builds/releases/download/v1.8.8-stable/synergy-v1.8.8-stable-Windows-x86.msi"
```
Comment (ferventcoder) [inline manual/synergy/tools/chocolateyInstall.ps1]: What is this url? Official?

Comment (ferventcoder) [inline manual/synergy/tools/chocolateyInstall.ps1]: We typically only allow sanctioned builds. If it is not listed from synergy.com, it's unlikely that we would be able to point here.

Comment (ferventcoder) [inline manual/synergy/tools/chocolateyInstall.ps1]: Sorry, https://symless.com/synergy

**Comment by gep13** [comment]
This package is now being maintained in this repository:

https://github.com/teknowledgist/Chocolatey-packages/tree/master/discontinued/synergy

As such, I am going to go ahead and close out this PR.

**Pull request closed without merging**